update load interited widget code lab (#94)
flutter/codelabs: dartpad_codelabs/src/inherited_widget/intro/instructions.md

# Introduction

> State management has been a really important topic in mobile development.

Reviewer: ```suggestion
State management is an important topic in mobile development.
```

> In Flutter, We provide the InheritedWidget aim to make the process of building

Reviewer: ```suggestion
In Flutter, InheritedWidget makes the process of building
```

> the state management system easier for our developers. In this workshop,

Reviewer: ```suggestion
the state management system easier. In this workshop,
```

> you will learn how to handle your app state by using the InheritedWidget.

Reviewer: ```suggestion
you will learn how to handle your app state by using InheritedWidget.
```

# An Overview

This workshop includes a series of steps to convert an app without
any state management to a fully centralized state management system.

> You will be started with a demo Google store app. To see how the app looks
> like, click the `Run` button on the top right corner of the IDE.

Reviewer: ```suggestion
You will be started with a demo Google store app. To see how the app looks,
click the `Run` button in the top right corner of the IDE.
```

Try to scroll through different google products and add/remove products from
the cart. The small cart icon on the right of the app bar changes based on number
of the products in the cart. You can also perform search queries by clicking the
search icon in the app bar and type `"nest"` to filter nest products.

> There are two components that require states:

Reviewer: ```suggestion
There are two components that require stateful objects:
```
